### ISSUE TITLE
Feature request: Manage BLAS symlinks

### ISSUE THREAD

**Issue by noamross** [r-lib/rig#212]
It would be very helpful if rig could take care of managing BLAS/LAPACK libraries used by all the installed versions by setting the relevant symlinks. There's a lot of payoff to switching these to Accelerate on macOS but it's a pretty obscure thing to set and track the status of. 

https://cran.r-project.org/bin/macosx/RMacOSX-FAQ.html#Which-BLAS-is-used-and-how-can-it-be-changed_003f

Something like `rig system blas set [version] [blas lib]` and `rig system blas status` to switch and look up the library of the R framework would be very helpful, or even an option to set the default for all R versions, including on install.

**Comment by westat-schneider** [comment]
Yes, this would be a godsend, even it was just limited to supporting Accelerate and OpenBLAS. Changing the BLAS/LAPACK libraries used by R is something that many users don't know how to do, but would nonetheless greatly benefit from. It would be nice if rig could simplify this process for users.
